docker/compose-controller-spark-sql.yaml

```diff
@@ -56,7 +56,7 @@
 # to /opt/bitnami/spark/conf/
 # https://spark.apache.org/docs/latest/configuration.html
 
-version: '2'
+version: '2.4'
```

Comment:
nit: consider adding a comment here why we needed this version.

Reply:
done